docs/quickstart-examples.md

```diff
@@ -0,0 +1,105 @@
+# Quickstart Examples
+
+Now that you have installed the basic requirements, you can start pulling data. Try out the examples below, or navigate to the [api/client/samples/](https://github.com/gro-intelligence/api-client/blob/development/api/client/samples) folder for more options.
+
+## Get Started in 120 Seconds
+
+If you have a google account, or if you already have jupyter notebooks and python installed, and a Gro API token, you can start pulling Gro data via our API client within 120 seconds.
+
+__Step 1: Get your API token__
+1. Login to your Gro web app account at [app.gro-intelligence.com](app.gro-intelligence.com)
+2. Open the Account button in the bottom left corner of the app.
+
+![account-button](./_images/account-button.PNG)
+
+3. Navigate to the API tab.
+
+![account-api-section](./_images/account-api-section.PNG)
+
+4. Copy your token.
+
+__Step 2: Run the code in a notebook__
+1. Navigate to [Google Colab](https://colab.research.google.com/), or launch jupyter notebooks from your command line with the command `jupyter notebook`. 
+
+2. Copy the below code and paste it into the first cell.
+```
+!pip install git+https://github.com/gro-intelligence/api-client.git
+```
+
+Then click the "Run Cell" button to the left of the cell.
+
+In Colab, it will look like this:
+
+![colab-run-button](./_images/colab-run-button.PNG)
+
+In Jupyter Notebook, it looks like this:
+
+![jupyter-run-button](./_images/jupyter-run-button.PNG)
+
+3. Add another cell with the "+ Code" button, then copy the below code and paste it into the second cell.
+```
+from api.client.gro_client import GroClient
+
+API_HOST = 'api.gro-intelligence.com'
+ACCESS_TOKEN = 'YOUR_TOKEN'
+client = GroClient(API_HOST, ACCESS_TOKEN)
+```
+Replace the text YOUR_TOKEN with the token you copied from step one. Then click the "Run Cell" button to the left of the cell.
+
+4. Add one more cell with the "+ Code" button, then copy the below code and paste it into the third cell.
+```
+# Rice, milled - Area Harvested (area) - Kenya (USDA PS&D)
+client.get_data_points(**{
+	'metric_id': 570001, 
+	'item_id': 392, 
+	'region_id': 1107, 
+	'source_id': 14, 
+	'frequency_id': 9
+})
+```
+Click the "Run Cell" button to the left of the cell, and that's it! You should now see the data in the response.

```

Comment:
Could include a screenshot at the end here of the complete notebook with the outputs

Reply:
Done